include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -195,6 +193,19 @@ class format_embl
         ++stream_it;
     }
 
+    /*!\brief Writes a number to the stream iterator using std::to_chars.
+     * \tparam stream_iterator_type The type of ostream iterator.
+     * \tparam number_type The type of number; must model seqan3::arithmetic.
+     * \param[in,out] stream_it The ostream iterator to write to.
+     * \param[in] num The number to write.
+     */
+    template <typename stream_iterator_type, arithmetic number_type>
+    auto write_number(stream_iterator_type & stream_it, number_type num)
```

Comment:
missing test cases

Reply:
now done in iterator test since `write_number` is now a member function of the iterator